index.js

```diff
@@ -38,6 +38,19 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('pre-id', {
```

Comment:
This should be call "tag", as https://docs.npmjs.com/cli/publish

Also you need to change the final `npm publish` information